mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, db_uri):
         self.db_uri = db_uri
         self.db_type = extract_db_type_from_uri(db_uri)
         self.engine = mlflow.store.db.utils.create_sqlalchemy_engine_with_retry(db_uri)
-        Base.metadata.create_all(self.engine)
```

Comment:
We should not initialize DB with `Base.metadata.create_all`.

Reply:
When we use the same DB for tracking and model registry, `TrackingSqlAlchemyStore.__init__` initializes the tables first so this line is a no-op. However, when we use separate DBs for tracking and model registry, this line attempts to create the tables.